src/Examine.Lucene/ExamineTaxonomyReplicator.cs

```diff
@@ -173,6 +183,7 @@ public void Dispose()
         {
             // Do not change this code. Put cleanup code in 'Dispose(bool disposing)' method
             Dispose(disposing: true);
+            GC.SuppressFinalize(this);
```

Comment:
Might have been my IDE giving me a warning. I have some extra warnings running. We can remove this again but I believe this follows the disposable pattern better which is also the reason for the out commented code in the LuceneSeacher as I remember. We can also remove this change again as it might not be relevant in the PR

Reply:
Removed